tests/e2e/manifest/consumer.yaml

```diff
@@ -0,0 +1,171 @@
+# Example consumer
```

Comment:
Do you need this? operator test should just verify the component it deploys are in running status and the  operand is running fine

Reply:
done. removed.